src/globalvars.cpp

```diff
@@ -220,6 +255,20 @@ SizedType GlobalVars::get_sized_type(const std::string &global_var_name,
                         CreateArray(resources.max_map_key_size, CreateInt8()));
   }
 
+  if (config.type == Type::integer) {
```

Comment:
Same as above, use `switch`? I think that would allow to get rid of the unused `return CreateInt64()` at the end.

Reply:
Mentioned this above but most of the types are unused.